CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Release 1.34.0 20-05-2019
+
+- Update googleapis to official repo (#621)
```

Comment:
remove this line,  it is not a feature or bug fixing.

Reply:
Done.